samples/sample_plugins/vpp_plugin/src/mfx_vpp_plugin.cpp

```diff
@@ -93,21 +93,22 @@ MFXVideoVPPPlugin::~MFXVideoVPPPlugin(void)
 }
 
 // per-component methods currently not implemented, you can add your own implementation if needed
-mfxStatus MFXVideoVPPPlugin::Query(mfxVideoParam *in, mfxVideoParam *out, mfxU8 component_idx)
+mfxStatus MFXVideoVPPPlugin::Query(mfxVideoParam *in, mfxVideoParam *out)
```

Comment:
Better change to nameless parameters
MFXVideoVPPPlugin::Query(mfxVideoParam * /*in*/, mfxVideoParam * /*out*/)